src/Swarm/Game/Step.hs

```diff
@@ -1064,6 +1064,37 @@ execConst c vs s k = do
           }
       updateRobotLocation loc nextLoc
       return $ Out VUnit s k
+    Stride -> case vs of
+      [VInt d] -> do
+        -- Figure out where we're going
+        loc <- use robotLocation
+        orient <- use robotOrientation
+        let heading = orient ? zero
+
+        -- Excludes the base location.
+        let locsInDirection :: [Location]
+            locsInDirection =
+              take (min (fromIntegral d) maxStrideRange) $
```

Comment:
Should trying to `stride` more than the `maxStrideRange` be an error?  Currently, if you say e.g. `stride 65` it will just silently stride 64 instead.